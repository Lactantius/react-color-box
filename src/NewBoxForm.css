.NewBoxForm {
  display: grid;
  justify-content: center;
  font-size: 1rem;
  margin-top: 2rem;
}

.NewBoxForm input {
  min-width: 15rem;
  padding: 0.2rem;
  margin: 1rem;
  justify-self: center;
  font-size: 1rem;
}

.NewBoxForm button {
  padding: 0.5rem;
  font-size: 2rem;
  color: white;
  background-color: #3868ee;
  border: none;
  border-radius: 5px;
}

.NewBoxForm-color-picker {
  height: 3rem;
}
