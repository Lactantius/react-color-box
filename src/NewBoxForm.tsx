import React from "react";

function Box(): JSX.Element {
  return <div className="Box">Box</div>;
}

export default Box;
