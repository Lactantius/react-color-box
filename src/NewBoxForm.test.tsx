import React from "react";
import { render, screen } from "@testing-library/react";
import NewBoxForm from "./NewBoxForm";

test("NewBoxForm renders", () => {
  render(<NewBoxForm />);
});
