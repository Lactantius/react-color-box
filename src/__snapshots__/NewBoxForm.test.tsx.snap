// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form
    class="NewBoxForm"
  >
    <label
      for="width"
    >
      Box Width
    </label>
    <input
      id="width"
      name="width"
      placeholder="width"
      type="number"
      value="0"
    />
    <label
      for="height"
    >
      Box Height
    </label>
    <input
      id="height"
      name="height"
      placeholder="height"
      type="number"
      value="0"
    />
    <label
      for="color"
    >
      Box Color
    </label>
    <input
      class="NewBoxForm-color-picker"
      id="color"
      name="color"
      placeholder="color"
      type="color"
      value="#000000"
    />
    <button>
      New Box
    </button>
  </form>
</DocumentFragment>
`;
